charts/gardener-extension-provider-alicloud/templates/clusterfilters-logging.yaml

```diff
@@ -1,47 +1,3 @@
-apiVersion: fluentbit.fluent.io/v1alpha2
-kind: ClusterFilter
-metadata:
-  labels:
-    fluentbit.gardener/type: "seed"
-    fluentbit.gardener.cloud/type: "seed"
-  name: cloud-controller-manager-alicloud-cloud-controller-manager
-spec:
-  filters:
-  - parser:
-      keyName: log
-      parser: kube-apiserver-parser
-      reserveData: true
-  match: kubernetes.*cloud-controller-manager*alicloud-cloud-controller-manager*
```

Comment:
Should this be removed since it is about the CCM ?

Reply:
Oops :)